bpaotu/bpaotu/query.py

```diff
@@ -234,6 +234,8 @@ def matching_samples(self):
         q = self._session.query(SampleContext)
         subq = self._build_taxonomy_subquery()
         q = self._apply_filters(q, subq).order_by(SampleContext.id)
+        return q
+        return q.all()
```

Comment:
this line doesn't do anything, it's behind another `return` - remove

Reply:
Fixed.